docs/development/writingzeppelininterpreter.md

```diff
@@ -69,6 +69,9 @@ Here is an example of `interpreter-setting.json` on your own interpreter.
         "defaultValue": "property2DefaultValue",
         "description": "Property 2 description"
       }, ...
+    },
+    "editor": {
```

Comment:
Good point! If the language is not specified **plain text mode** will be used for syntax highlight, which will highlight nothing. I added description in c7221b2. Thanks for review :)